### DISCUSSION TITLE
openpdf1.3.34version ，How can I set and get the width and height of a paragraph? If it's not possible directly, are there any alternative solutions?   thanks

### DISCUSSION THREAD

**Discussion by JUNE-Colin** [LibrePDF/OpenPDF#1155] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Your real name**
Please specify your full name here, so that we can verify your identity. 
If you have a conflict of interest describe this here also.

**Additional context**

**Comment by asturio** [comment]
Hi @JUNE-Colin ,

can you please describe in the content (the template was untouched), what you  tried, and what do you want to achieve?
